src/main/java/me/totalfreedom/totalfreedommod/util/FUtil.java

```diff
@@ -53,12 +57,32 @@
             ChatColor.LIGHT_PURPLE,
             ChatColor.YELLOW);
 
+    public static final List<ChatColor> CHAT_RAINBOW = Arrays.asList(
```

Comment:
No, I ran /rainbownick Ryan about 6 times, and /rainbownick Wild1145 about 3 times, I even sent you the video of it. Your command doesnt work.